packages/utils/src/date/index.ts

```diff
@@ -175,6 +197,26 @@ export function timeZoneExists(timeZone: string): boolean {
 	}
 }
 
+const locales: Map = { fr, ja, de, en: enUS };
+
+export const localizedFormat = (t: any) => ({
+	MDY_LONG: t('DATE_FORMAT_MONTH_DAY_YEAR_LONG', { defaultValue: 'MMMM do, yyyy' }), // June 25th, 2021
+	MY_LONG: t('DATE_FORMAT_MONTH_YEAR_LONG', { defaultValue: 'MMMM yyyy' }), // June 2021
+	MDY: t('DATE_FORMAT_MONTH_DAY_YEAR', { defaultValue: 'MM/dd/yyyy' }), // 06/25/2021
+	MDYHM: t('DATE_FORMAT_MONTH_DAY_YEAR_HOUR_MIN', { defaultValue: 'MM/dd/yyyy hh:mm aaa' }), // 06/25/2021 11:44 am
```

Comment:
List to be updated will all needed date formats before asking the translation team for localized patterns.

Reply:
May be we can rely on Intl for that